Networking/Networking/Model/Product/ProductVariation.swift

```diff
@@ -277,6 +277,9 @@ public struct ProductVariation: Codable, GeneratedCopiable, Equatable, Generated
 
         try container.encode(image, forKey: .image)
 
+        // Needed for bulk update variations, in other API calls it is ingored
+        try container.encode(productVariationID, forKey: .productVariationID)
```

Comment:
Could you explain why is this needed now?

Reply:
The API Call for the bulk update sends as payload an array of product variations. The server expects for each object in the array to have an ID (so that it knows that variation to update).
Example payload:

```
{
  "update": [
    {
      "id": 733,
      "regular_price": "10.00",
....
    }
  ]
}
```

For other API calls that do not need the variation ID, this extra value is send, but the server ignores it. An other strategy could be to add an other property , something like `encodedProductVariationID`, that would be encoded if not nil, and set it to the value productVariationID has before the bulk update API call. But I though it was adding complexity without offering much.

Reply:
I see now, this should be safe then. 
Thanks for the explanation!